<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GCDestructiveEditDialog</class>
 <widget class="QDialog" name="GCDestructiveEditDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>609</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Remove Items from Profile</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/gcresources.qrc">
    <normaloff>:/resources/goblinicon.jpg</normaloff>:/resources/goblinicon.jpg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog
{
  background-color: rgba(150, 150, 150, 255);
}

QPushButton
{
  color: rgb(0, 0, 0);
  background-color: qlineargradient(spread:reflect, x1:0.5, y1:0.5, x2:0.5, y2:1, stop:0 rgba(110, 110, 110, 240), stop:1 rgba(150, 150, 160, 255));
  border-width: 1px;
  border-radius: 5px;
  border-color: gray;
  border-style: outset;
  font: 12px;
  min-width: 5em;
  min-height: 1em;
  padding: 3px;
}

QPushButton:pressed
{
  color: rgb(190, 190, 190);
  background-color: qlineargradient(spread:reflect, x1:0.5, y1:0.5, x2:0.5, y2:1, stop:0 rgba(150, 150, 160, 240), stop:1 rgba(110, 110, 125, 255));
  border-style: inset;
}

QPushButton:default 
{
  border-color: gray;
  border-width: 2px;   
  background-color: qlineargradient(spread:reflect, x1:0.5, y1:0.5, x2:0.5, y2:1, stop:0 rgba(110, 110, 110, 255), stop:1 rgba(190, 190, 190, 255));
}

QToolButton
{
  background-color: qlineargradient(spread:reflect, x1:0.5, y1:0.5, x2:0.5, y2:1, stop:0 rgba(110, 110, 110, 255), stop:1 rgba(150, 150, 160, 255));
  border-width: 1px;
  border-radius: 5px;
  border-color: rgb(40, 40, 40);
  border-style: outset;
}

QToolButton:pressed
{
  background-color: qlineargradient(spread:reflect, x1:0.5, y1:0.5, x2:0.5, y2:1, stop:0 rgba(110, 110, 110, 255), stop:1 rgba(150, 150, 160, 255));
  border-style: inset;
}

QComboBox
{
  color: rgb(250, 250, 250 );
  background-color: qlineargradient(spread:reflect, x1:0.5, y1:0.5, x2:0.5, y2:1, stop:0 rgba(110, 110, 110, 240), stop:1 rgba(150, 150, 160, 255));
  selection-background-color: rgb(90, 90, 90);
  border-width: 1px;
  border-radius: 5px;
  border-color: rgb(40, 40, 40);
  border-style: outset;
  font: 12px;
  min-width: 7em;
  min-height: 1em;
  padding: 2px;
  text-align: right;
}

QComboBox::drop-down
{
  subcontrol-origin: padding;
  subcontrol-position: top right;
  left: -5px;
  width: 18px;
  border-left-color: transparent;
  text-align: right;
}

QComboBox::down-arrow
{
  width: 17px;
  height: 17px;
  image: url(&quot;:/resources/downarrow.png&quot;);
  opacity: 0.1;
}

QComboBox QAbstractItemView
{
  selection-color: rgb(20, 20, 20);
  color: rgb(200, 200, 200);
  selection-background-color: rgba(110, 110, 110, 255);
}

QComboBox:disabled
{
  color: rgb(0, 0, 0);
  background-color:rgba(110, 110, 110, 255);
}

QListView
{
  border-width: 1px;
  border-radius: 5px;
  border-color: rgb(40, 40, 40);
  border-style: inset;
  background-color: qlineargradient(spread:pad, x1:0.767, y1:1, x2:1, y2:0, stop:0 rgba(40, 40, 40, 200), stop:1 rgba(120, 120, 120, 255));
  font: 11px;
  color: rgb(240, 240, 240);
}

QLabel
{
  background-color: rgba(140, 140, 140, 200);
  border-radius: 6px;
  padding: 1px;
  font: 12px;
}

QLineEdit
{
  border-width: 1px;
  border-radius: 5px;
  border-color: rgb(90, 90, 90);
  border-style: solid;
  padding: 2 4px;
  background-color: rgb(180, 180, 180);
  selection-background-color: rgb(98, 96, 122);
  font: 12px;
}

QLineEdit:disabled
{
  color: rgb(0, 0, 0);
  background-color:rgba(110, 110, 110, 255);
}

QPlainTextEdit
{
  background-color: lightgray;
  selection-background-color: rgba(120, 120, 130, 200);
}

QHeaderView::section 
{
  color: rgba(10, 10, 10, 255);
  background-color: qlineargradient(spread:reflect, x1:0.5, y1:0.5, x2:0.5, y2:1, stop:0 rgba(110, 110, 110, 240), stop:1 rgba(150, 150, 160, 255));
  border-width: 1px;
  border-radius: 3px;
  border-color: darkgray;
  border-style: inset;
  font: 11px;
  min-width: 5em;
  min-height: 1em;
  padding: 3px;
}

QHeaderView::section:checked
{
  alternate-background-color: rgba(90, 90, 100, 255);
}

QHeaderView::down-arrow 
{
  image: url(&quot;:/resources/downarrow.png&quot;);
}

QHeaderView::up-arrow 
{
  image: url(&quot;:/resources/uparrow.png&quot;);
}

QTreeView
{
  border: 1px solid black;
  color: rgba(10, 10, 10, 255);
  background-color: rgba(140, 140, 140, 255);
  alternate-background-color: rgba(90, 90, 100, 255);
  selection-background-color: rgba(120, 120, 130, 200);
  font: 12px;
}

QTreeView::branch:has-children:!has-siblings:closed, QTreeView::branch:closed:has-children:has-siblings
{
  border-image: none;
  image: url(&quot;:/resources/rightarrow.png&quot;);
}

QTreeView::branch:open:has-children:!has-siblings, QTreeView::branch:open:has-children:has-siblings
{
  border-image: none;
  image: url(&quot;:/resources/angledarrow.png&quot;);
}

QTreeView::branch:hover
{
  background-color: rgba(20, 20, 20, 200);
}

QTreeView::item:selected 
{
   color: rgba(20, 20, 20, 255);
}

QScrollBar:vertical 
{
  width: 17px;
  margin: 18px 0 18px 0;
  background-color: qlineargradient(spread:pad, x1:0.767, y1:1, x2:1, y2:0, stop:0 rgba(140, 140, 140, 200), stop:1 rgba(160, 160, 160, 255));
  border-width: 1px;
  border-color: rgb(40, 40, 40);
  border-style: solid;
  border-radius: 3px;
}

QScrollBar::handle:vertical 
{
  background-color: qlineargradient(spread:reflect, x1:0, y1:1, x2:0, y2:0.5, stop:0 rgba(50, 50, 50, 255), stop:1 rgba(150, 150, 150, 255));
  width: 17px;
  height: 17px;
  image: url(&quot;:/resources/scrollgrip.png&quot;);
  min-height: 3em;
  border-color: rgb(40, 40, 40);
  border-style: solid;
  border-radius: 3px;
}

QScrollBar::add-line:vertical 
{
  border-color: rgba(40, 40, 40, 0);
  height: 18px;
  subcontrol-position: bottom;
  subcontrol-origin: margin;
}

QScrollBar::sub-line:vertical 
{
  border-color: rgba(40, 40, 40, 0);
  height: 18px;
  subcontrol-position: top;
  subcontrol-origin: margin;
}

QScrollBar::up-arrow:vertical
{
  width: 17px;
  height: 17px;
  image: url(&quot;:/resources/uparrow.png&quot;);
  background-color: qlineargradient(spread:pad, x1:0.767, y1:1, x2:1, y2:0, stop:0 rgba(110, 110, 125, 255), stop:1 rgba(150, 150, 160, 255));
  border-radius: 2px;
}

QScrollBar::up-arrow:vertical:pressed
{
  color: rgb(190, 190, 190);
  background-color: rgba(50, 50, 50, 255);
  border-style: inset;
}

QScrollBar::down-arrow:vertical 
{
  width: 17px;
  height: 17px;
  image: url(&quot;:/resources/downarrow.png&quot;);
  background-color: qlineargradient(spread:pad, x1:0.767, y1:1, x2:1, y2:0, stop:0 rgba(110, 110, 125, 255), stop:1 rgba(150, 150, 160, 255));
  border-radius: 2px;
}

QScrollBar::down-arrow:vertical:pressed
{
  color: rgb(190, 190, 190);
  background-color: rgba(50, 50, 50, 255);
  border-style: inset;
}

QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical 
{
  background: none;
}

QScrollBar:horizontal 
{
  height: 17px;
  margin: 0 18px 0 18px;
  background-color: qlineargradient(spread:pad, x1:0.767, y1:1, x2:1, y2:0, stop:0 rgba(140, 140, 140, 200), stop:1 rgba(160, 160, 160, 255));
  border-width: 1px;
  border-color: rgb(40, 40, 40);
  border-style: solid;
  border-radius: 3px;
}

QScrollBar::handle:horizontal 
{
  background-color: qlineargradient(spread:reflect, x1:0, y1:1, x2:0, y2:0.5, stop:0 rgba(50, 50, 50, 255), stop:1 rgba(150, 150, 150, 255));
  width: 17px;
  height: 17px;
  image: url(&quot;:/resources/scrollgriphorizontal.png&quot;);
  min-width: 3em;
  border-color: rgb(40, 40, 40);
  border-style: solid;
  border-radius: 3px;
}

QScrollBar::add-line:horizontal 
{
  border-color: rgba(40, 40, 40, 0);
  width: 18px;
  subcontrol-position: right;
  subcontrol-origin: margin;
}

QScrollBar::sub-line:horizontal 
{
  border-color: rgba(40, 40, 40, 0);
  width: 18px;
  subcontrol-position: left;
  subcontrol-origin: margin;
}

QScrollBar::left-arrow:horizontal
{
  width: 17px;
  height: 17px;
  image: url(&quot;:/resources/leftarrow.png&quot;);
  background-color: qlineargradient(spread:pad, x1:0.767, y1:1, x2:1, y2:0, stop:0 rgba(110, 110, 125, 255), stop:1 rgba(150, 150, 160, 255));
  border-radius: 2px;
}

QScrollBar::left-arrow:horizontal:pressed
{
  color: rgb(190, 190, 190);
  background-color: rgba(50, 50, 50, 255);
  border-style: inset;
}

QScrollBar::right-arrow:horizontal 
{
  width: 17px;
  height: 17px;
  image: url(&quot;:/resources/rightarrow.png&quot;);
  background-color: qlineargradient(spread:pad, x1:0.767, y1:1, x2:1, y2:0, stop:0 rgba(110, 110, 125, 255), stop:1 rgba(150, 150, 160, 255));
  border-radius: 2px;
}

QScrollBar::right-arrow:horizontal:pressed
{
  color: rgb(190, 190, 190);
  background-color: rgba(50, 50, 50, 255);
  border-style: inset;
}

QScrollBar::add-page:horizontal, QScrollBar::sub-page:horizontal 
{
  background: none;
}

QSplitter::handle
{
  background-color: darkgray
}

QSplitter::handle:vertical 
{
  height: 3px;
}

QSplitter::handle:pressed
{
  background-color: lightgray
}
</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="elementLayout">
       <item>
        <widget class="QTreeWidget" name="treeWidget">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="indentation">
          <number>10</number>
         </property>
         <column>
          <property name="text">
           <string>Element Hierarchy</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="deleteElementLayout">
         <item>
          <spacer name="elementSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="removeChildButton">
           <property name="text">
            <string>Remove Child</string>
           </property>
           <property name="default">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="deleteElementButton">
           <property name="text">
            <string>Delete Element</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="elementHelpButton">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="../resources/gcresources.qrc">
             <normaloff>:/resources/helpcursor.png</normaloff>:/resources/helpcursor.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="attributeLayout">
       <item>
        <widget class="QComboBox" name="comboBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="plainTextEdit"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="deleteAttributeLayout">
         <item>
          <spacer name="attributeSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="updateValuesButton">
           <property name="text">
            <string>Update Values</string>
           </property>
           <property name="default">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="deleteAttributeButton">
           <property name="text">
            <string>Delete Attribute</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="attributeHelpButton">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="../resources/gcresources.qrc">
             <normaloff>:/resources/helpcursor.png</normaloff>:/resources/helpcursor.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources/gcresources.qrc"/>
 </resources>
 <connections/>
</ui>
